ntbs-service/Services/LogService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Serilog;
+
+namespace ntbs_service.Services
+{
+    public interface ILogService
+    {
+        void LogWarning(string message);
+    }
+    
+    public class LogService : ILogService
```

Comment:
It might be worth adding a comment to this class explaining why we have it. It may seem strange to devs that in some places we're calling Serilog directly, and in other places we use this service (however, I don't think it's worth going through the code and updating all the places Serilog is called!). So perhaps a comment like "This class is a thin wrapper around Serilog which we use when we want to test the logging behaviour" would be appropriate.